src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -48,7 +48,7 @@ public function __construct(
     }
 
     /**
-     * @Given the store ships everything for free within the :zone zone
```

Comment:
why?

Reply:
https://github.com/Sylius/Sylius/blob/1.13/features/order/managing_orders/order_details/seeing_order_item_taxes.feature#L13

It turned out that IDE IntelliSense is case-sensitive and was considering the step as not implemented.

Reply:
So I guess we should change the scenario's step 🤔. "Free" with uppercased letter looks silly here.